windows_build.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2020, 219 Design, LLC
+# See LICENSE.txt
+#
+# https://www.219design.com
+# Software | Electrical | Mechanical | Product Design
+#
+
+set -Eeuxo pipefail # https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+
+pushd .
+trap "popd" EXIT HUP INT QUIT TERM
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd $DIR  # enter this script's directory. (in case called from root of repository)
+
+DL_FOLDER=$DIR/dl_third_party
+
+pip3 install aqtinstall  # https://github.com/miurahr/aqtinstall
+
+# NOTE: as of Nov 23, 2020, it is not clear whether 'win64_msvc2019_64' is the right ARCH
+# argument to pass to aqtinstall. We may need to try other options:
+#   win64_msvc2017_64, win64_msvc2019_winrt_x64 ?
+# Refer to: https://github.com/miurahr/aqtinstall#usage
+
+# https://github.com/miurahr/aqtinstall/issues/126 "Installing smaller subset of the libraries"
+python -m aqt install --outputdir $DL_FOLDER/Qt_desktop 5.15.0 windows desktop win64_msvc2019_64 --archives \
+        icu \
+        qtbase \
+        qtconnectivity \
+        qtdeclarative \
+        qtgraphicaleffects \
+        qtimageformats \
+        qtquickcontrols \
+        qtquickcontrols2 \
+        qtsvg \
+        qttools
+
+$DIR/tools/ci/version.sh
+
+echo "test5"
+exit 0
```

Comment:
Everything succeeds up til this (intentional early) exit.

The goal going forward is ("in so many words") to remove this `exit 0` (and the spurious `echo` above it), and do whatever other changes are needed to make the REST of this bash file succeed.